python/pyarrow/io.pxi

```diff
@@ -1670,16 +1672,16 @@ def output_stream(source, compression='detect', buffer_size=None):
 
     Parameters
     ----------
-    source: str, Path, buffer, file-like object, ...
-        The source to open for writing
+    source: str, pathlib.Path, buffer, file-like object, ...
```

Comment:
Keep "Path" as we're really talking about path-like objects in general (any object with a `__fspath__` method should do).

Reply:
Fixed.